jest.setup.js

```diff
@@ -9,6 +9,67 @@ jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');
 
 jest.mock('@datadog/mobile-react-native');
 
+// Mock MMKV
```

Comment:
Why are these changes needed?

Also, see - [How to write better code comments](https://dev.to/adammc331/todo-write-a-better-comment-4c8c)